### ISSUE TITLE
Unify modals "close" button

### ISSUE THREAD

**Issue by szymonlesisz** [trezor/trezor-suite#5618]
Close buttons have inconsistent layout in different modals. It would be nice to have them displayed in the same way everywhere.

Related issue: https://github.com/trezor/trezor-suite/issues/5540

Examples:

![Screenshot from 2022-06-15 11-02-31](https://user-images.githubusercontent.com/3435913/173809793-214d76af-3b38-4e50-afbd-52e40ceb2ff8.png)

![Screenshot from 2022-06-15 11-03-28](https://user-images.githubusercontent.com/3435913/173809775-ad79ff4b-1506-4084-83a8-209e9b8e2495.png)

![Screenshot from 2022-06-15 11-02-52](https://user-images.githubusercontent.com/3435913/173809787-bb9eccda-9e4a-47f2-9c6a-d14add33296e.png)

**Comment by dahaca** [comment]
I'd say we use the **"abort"** button from Image 2 in all device prompt modals which can are cancelable. So far it was only done for generic device prompts, the **"follow the instruction"** ones. It can now be extended to more specific ones, like the **"send"** one from the Image 1. The modal on Image 3 does not interact with the device so it should remain as is.

**Comment by komret** [comment]
@dahaca I context of the current redesign, is there anything to do, or should we close this issue?

**Comment by dahaca** [comment]
@komret I would say not close but restructure: the issue Szymon pointed to is that the close button is in different places, however I would argue that the [`AbortButton`](https://github.com/trezor/trezor-suite/blob/7d0bb2198f8b67d76b82658a2285fbef65e68c09/packages/suite/src/components/suite/modals/AbortButton.tsx#L75) is essentially like the close button with additional context – the word "Abort". Currently it does not inherit its style from the `IconButton` global component, which it eventually should. And I never was a fan of the `cancel decive request` button located in the "device pill" but that is out of the scope of the redesign, as it's a very comprehensive issue in itself. 🙏 

TLDR: we _should eventially_ restyle the `AboutButton` to make it looks like the `IconButton` – this is scoped in. "Device pill" logic – not scoped in. If you agree, please rename the issue and add it to the milestone ✌️

**Comment by komret** [comment]
@dahaca
- Using `IconButton` in `AccountButton` sounds reasonable. The default `Modal` close button looks different in the new design so this will solve the original issue of the close button looking differently in image 2 and image 3. I can create an issue for that.
- This won't solve the fact that the abort button is sometimes in the pill, sometimes in the modal, which is obviously wrong. Fixing it will resolve this issue. However, we need a decision where the button should be. (See image 1 vs image 2 above).

Agreed?

**Comment by dahaca** [comment]
@komret solving the device indicators is not a part of the first phase of empower 🙏  But sure, polishing the about Button is very much doable!

**Comment by komret** [comment]
Resolved by Empower except for https://github.com/trezor/trezor-suite/issues/14260

**Comment by bosomt** [comment]
**QA OK**


**Info:**
 - Suite version: web 24.10.0 (d48f1d337c96e29fad684a19fbb55bd842b9df74)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:130.0) Gecko/20100101 Firefox/130.0
 - OS: MacIntel
 - Screen: 1512x982
 - Device: Trezor T2T1 2.8.1 regular (revision 632b9561559b7ab6824bb7eeac072874e07b7b82)
 - Transport: BridgeTransport 3.0.0-bundled.24.10.0
